dbt/adapters/databricks/impl.py

```diff
@@ -769,21 +769,28 @@ def generate_unique_temporary_table_suffix(self, suffix_initial: str = "__dbt_tm
 
     @available
     @staticmethod
-    def get_enriched_columns(
-        existing_columns: list[DatabricksColumn], model_columns: dict[str, dict[str, Any]]
-    ) -> list[DatabricksColumn]:
+    def parse_columns_and_constraints(
```

Comment:
Is this used anywhere?

Reply:
Only in my branch where I found out it doesn't really work.  I tried to isolate model and column constraint parsing/processing before, but when I tried to use the constraint code on my local branch, I discovered that Databricks does not like multiple column constraints on a single column; also dbt supports specifying a bunch of null constraints at the model level.  Hence why I have to do this less elegant, 'process them all at the same time' approach :/.